{
  try {
    return update(varname,value);
  }
 catch (  EvalException ee) {
    throw new AssertionError(ee);
  }
}
